taglib/src/test/java/com.reevoo.taglib/ReevooTaglibTest.java

```diff
@@ -24,7 +24,7 @@ public void setUp() throws Exception {
         super.setUp();
         reevooTag.setDynamicAttribute("", "sku", "ABC123");
         reevooTag.setTrkref("FOO");
-        reevooTag.setBaseURI("http://mark.reevoo.com/endpoint");
+        reevooTag.setBaseURI("https://mark.reevoo.com/endpoint");
```

Comment:
This is public content anyway I don't think we should worry that much about security but as long as it redirects to https that it's fine to have it hardcoded to https.